src/find-options/FindOptionsUtils.ts

```diff
@@ -391,7 +391,7 @@ export class FindOptionsUtils {
                 qb.connection.driver,
                 { joiner: "__" },
                 alias,
-                relation.propertyPath,
+                relation.propertyName,
```

Comment:
@felix-gohla 
This was the main issue, breaking nested relations. There was not really the need of  parsing "." to distinguish relations

Reply:
This is great! 😊 Simple solutions are always the best!